float32 J1
float32 J2
float32 J3
float32 J4
float32 J5
float32 J6
float32 TCP_X
float32 TCP_Y
float32 TCP_Z
float32 Roll
float32 Pitch
float32 Yaw
uint8 Robot_Reach_Flag
uint8 DO0
uint8 DO1
uint8 DO2
uint8 DO3
uint8 DO4
uint8 DO5
uint8 DO6
uint8 DO7
